### ISSUE TITLE
No separate option to enable dark mode 

### ISSUE THREAD

**Issue by MithNo1** [calcom/cal.com#11997]
### Is your proposal related to a problem?
Yes
  

(Write your answer here.)
So almost every website or saas these days have a separate button within there UI to enable dark mode it's a very small but big thing that effects usability for the service because let's say I only want dark mode in notion or cal.com but not on twitter or Instagram. By default it would force dark mode if I changed my device theme to dark and all other websites where I don't want to see dark mode but without doing it cal.com wouldn't change to dark mode which is a problem and I think it should fixed it's a small thing but does effect usability. 
### Describe the solution you'd like
Add a separate option to change cal.com theme to dark mode in settings or main UI without changing my device theme. 

(Describe your proposed solution here.)
Written above
### Describe alternatives you've considered
There's no alternative whatsoever I have to change my device theme to dark then only it works no other way. 

(Write your answer here.)

### Additional context

(Write your answer here.)

### Requirement/Document

(Share it here.)

**Comment by ciaranha** [comment]
@sean-brydon is this easy for us to do now?

**Comment by PeerRich** [comment]
I agree, we can add an extra section here https://app.cal.com/settings/my-account/appearance

![CleanShot 2023-11-27 at 16 13 37@2x](https://github.com/calcom/cal.com/assets/8019099/960c93e1-18dc-423c-a46b-3398e583b256)
